docker/Dockerfile

```diff
@@ -17,33 +18,70 @@
 FROM openjdk:jre-alpine
```

Comment:
Use specific version of the image

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/966/commits/fe1f68c2c9d1d39e961d2664256249498d69c5aa